src/server/utilities/cspHeader/index.test.js

```diff
@@ -254,6 +276,8 @@ describe('cspHeader', () => {
         'https://tpc.googlesyndication.com',
         'https://gn-web-assets.api.bbc.com',
         'https://www.googletagservices.com',
+        'https://securepubads.g.doubleclick.net',
+        'https://bbc.gscontxt.net',
```

Comment:
Seems to be a duplication of line 264?

Reply:
This is for testing: isAmp: false, sLive: true